src/nvim/lua/executor.c

```diff
@@ -1417,20 +1427,20 @@ void ex_luafile(exarg_T *const eap)
 ///
 /// @param  path  path of the file
 ///
-/// @return  true if everything ok, false if there was an error (echoed)
+/// @return  false when unable to loadfile othewise return true
```

Comment:
LGTM, but maybe be explicit here that `false` is returned if opening or reading the file fails (to disambiguate it from the other things `loadfile` does).

Reply:
good catch . I actually forgot to change the comment after changing the code . Fixed now :)